index.html

```diff
@@ -0,0 +1,125 @@
+                <li>bootstrap<img src="./assets/images/vert-line.png" alt="vertical line"></li>  
+                <li>Ruby</li>
+            </ul>
+
+            <button type="button" class="project-button">See Project</button>
+
+        </section>
+        <section class="experience-container">
+            <img src="./assets/images/bg.jpg" alt="project image">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="stack-container">
+                <li>css <img src="./assets/images/vert-line.png" alt="vertical line"></li>
+                <li>html<img src="./assets/images/vert-line.png" alt="vertical line"></li>          
+                <li>bootstrap <img src="./assets/images/vert-line.png" alt="vertical line"></li>         
+                <li>Ruby</li>
+            </ul>
+            <button type="button" class="project-button">See Project</button>
+
+        </section>
+        <section class="item"></section>
+        <section class="experience-container">
+            <img src="./assets/images/bg.jpg" alt="project image">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="stack-container">
+                <li>css<img src="./assets/images/vert-line.png" alt="vertical line"></li>
+                <li>html<img src="./assets/images/vert-line.png" alt="vertical line"></li>              
+                <li>bootstrap<img src="./assets/images/vert-line.png" alt="vertical line"></li>
+                <li>Ruby</li>
+            </ul>
+            <button type="button" class="project-button">See Project</button>
+
+        </section>
+        <section class="item"></section>
+        <section class="experience-container">
+            <img src="./assets/images/bg.jpg" alt="project image">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="stack-container">
+                <li>css<img src="./assets/images/vert-line.png" alt="vertical line"></li>             
+                <li>htm<img src="./assets/images/vert-line.png" alt="vertical line">l</li>
+                <li>bootstrap<img src="./assets/images/vert-line.png" alt="vertical line"></li>              
+                <li>Ruby</li>
+            </ul>
+            <button type="button" class="project-button">See Project</button>
+
```

Comment:
- [ ] Please delete the extra blank line here
_For more details about this requirement, please see the comment above._